.gitignore

```diff
@@ -21,3 +21,9 @@ src/django/toga.min.js.map
 src/gtk/gi
 src/gtk/pygtkcompat
 .DS_Store
+pip-selfcheck.json
+pyvenv.cfg
+.vscode
+bin/
```

Comment:
I'd prefer to not `.gitignore` the `bin/` and `lib/` directories. We might have real things in those directories one day (especially lib) and I know if that day comes we're going to spend 20 minutes going "why the heck isn't this dir committing?" 😛